newton/examples/selection/example_selection_cartpole.py

```
@@ -174,13 +174,13 @@ def render(self):
         self.viewer.end_frame()
 
     def test(self):
-        num_bodies_per_env = self.model.body_count // self.num_envs
+        num_bodies_per_world = self.model.body_count // self.num_worlds
         newton.examples.test_body_state(
             self.model,
             self.state_0,
             "cart is at ground level and has correct orientation",
             lambda q, qd: q[2] == 0.0 and newton.utils.vec_allclose(q.q, wp.quat_identity()),
```

Ruff (0.14.0)

Unused lambda argument: `qd`

(ARG005)